src/types/response.rs

```diff
@@ -207,6 +190,81 @@ impl QueryResponse {
     }
 }
 
+#[derive(Debug, PartialEq, Clone)]
+#[allow(non_snake_case)]
+pub struct EntityHistoryElement {
+    pub db___valid_time: String,
+    pub tx___tx_id: usize,
+    pub tx___tx_time: String,
```

Comment:
Do you think at some moment and time we should deal with this in another type instead of `String`?

I don't think this is necessary for this Pull Request, but maybe we should think about this later :slightly_smiling_face: I don't know which date/time type Rust people usually use.

Reply:
Yes! it is my plan to create a feature to support `chrono`. Whats do you think?

Reply:
Hmm that's interesting, I've used before, but maybe it is bloated for what we need. We will see this later as we discussed outside here.